package.sh

```diff
@@ -7,17 +7,7 @@ echo Prune dependencies
 rm -rf node_modules packages/*/node_modules
 
 echo Install production dependencies
-if [[ $INTEGRATION_TEST =~ ^(true|1)$ ]]; then
```

Comment:
Is the `INTEGRATION_TEST` env still in use?

Reply:
removed